near-sdk/src/lib.rs

```diff
@@ -70,3 +73,6 @@ pub use serde;
 
 #[doc(hidden)]
 pub use serde_json;
+
+#[doc(hidden)]
+pub use schemars;
```

Comment:
let's just remove this altogether, re-exporting things like this is generally a bad pattern and the others would be purged if it wasn't very breaking of a change

Reply:
Noting again this would be ideal not to export and require that we keep this API stable

Reply:
Gotcha, moved behind `__private` for now since I can't figure out how to remove it